---
layout: post
research: true
category: misc
tags: []
---

<a name="research"></a>
# Our Research

## Framework

We typically work with ordinary differential equation (ODE) models derived from mass action kinetics as they provide efficient means to encode our knowledge of biological system, allow us to  make sense of dynamically resolved perturbation data and have direct biophysical interpretation. ODE models are well-established means to describe molecular pathways, but have not caught up with the advent of structural methods such as cryogenic electron microscopy, single molecule fluorescence resonance energy transfer or AlphaFold. These structural methods provide an unprecedentedly detailed (dynamic) picture of molecular mechanisms that is not faithfully represented by ODE models of molecular circuits, impeding interpretability and potentially diminishing predictive power. Similarly, omics methods such as proteomics and single-cell RNA sequencing offer a systems level perspective of the cell identity and its cell-to-cell variability, but this has not been connected to variability in the dynamics response of molecular pathways.

Our vision is to build models that bridge the structural, molecular, and systems level, encoding what we know about biologal systems in mathematical models and filling in the knowledge gaps using machine learning. We apply these models to interesting biological problems, including the prediction of sensitivity to anti-cancer drugs, anticipation of cell fate decisions and design of synthetic molecular switches.

## Research Interests