---
layout: post
research: true
category: misc
tags: []
---

<a name="research"></a>
# Our Research

## Motivation

We typically work with ordinary differential equation (ODE) models derived from mass action kinetics as they provide efficient means to encode our knowledge of biological system, allow us to  make sense of dynamically resolved perturbation data and have direct biophysical interpretation. ODE models are well-established means to describe molecular circuits, but have not caught up with the advent of structural methods such as cryogenic electron microscopy (cryo-EM), single molecule fluorescence resonance energy transfer (smFRET) or AlphaFold. These structural methods provide an unprecedentedly detailed (dynamic) picture of molecular mechanisms that is not faithfully represented by ODE models of molecular circuits, impeding interpretability and potentially diminishing predictive power. Similarly, omics methods such as proteomics and single-cell RNA sequencing offer a systems level perspective of the state of a cell and it's cell-to-cell variability, but models of molecular circuits usually explain cell-to-cell variability using stochastic frameworks that preclude data-driven predictions.

Our vision is to build models that bridge the structural, molecular, and systems worlds, encoding what we know about biologal systems and filling in the knowledge gaps using machine learning and apply them to interesting biological problems in cancer, developmental, and fundamental biology.

## Research Interests